maven/spec/dependabot/maven/requirement_spec.rb

```diff
@@ -74,6 +74,21 @@
     context "with a dynamic version requirement" do
       let(:requirement_string) { "1.+" }
       its(:to_s) { is_expected.to eq(Gem::Requirement.new("~> 1.0").to_s) }
+
+      context "that specifies a minimum" do
+        let(:requirement_string) { "1.5+" }
+        its(:to_s) { is_expected.to eq(Gem::Requirement.new("~> 1.5").to_s) }
+      end
+
+      context "that is just a +" do
+        let(:requirement_string) { "+" }
+        its(:to_s) { is_expected.to eq(Gem::Requirement.new(">= 0").to_s) }
+      end
```

Comment:
Copied these from the gradle requirement specs https://github.com/dependabot/dependabot-core/blob/main/gradle/spec/dependabot/gradle/requirement_spec.rb#L54-L67